src/stdlib/io.rs

```diff
@@ -9,19 +9,36 @@ use std::{io::Write, ops::Deref};
 fndr_native_func!(
     /// Prints to stdout
     print_func,
-    |_, item| {
+    |_, item, argv| {
         print!("{}", item.to_string());
-        let mut lock = std::io::stdout().lock();
-        let _ = lock.flush();
+        let argv = match &*argv {
+            List(l) => l.deref(),
+            _ => unreachable!(),
+        };
+        for item in argv {
+            print!("{}", item.to_string());
+        }
         Ok(item)
     }
 );
 
+const DEFAULT_ARGV: Vec<FenderReference> = Vec::new();
+
 fndr_native_func!(
     /// Prints to stdout and adds a newline
     println_func,
-    |_, item| {
-        println!("{}", item.to_string());
+    |_, item, argv| {
+        print!("{}", item.to_string());
+        let argv = match &*argv {
+            List(l) => l.deref(),
+            
+            _ => unreachable!(),
+        };
+        for item in argv {
+            print!("{}", item.to_string());
```

Comment:
Can we print a space between each element if multiple are passed? You should be able to just change this format string to `" {}"`. And same for the `print` function